Lib/Rabbit4000/AES_ENCRYPTION/AES_CRYPT.LIB

```diff
@@ -1638,10 +1650,13 @@ void _f_AESdecrypt4(AESstreamState __far * rk, char __far *data_in, char __far *
    ld		py,(pz+8)
    ld		pz,(pz+12)
 
+#if _RAB6K
+   altd ld		py,(sp+@sp+ek+4)

```

Comment:
Hmm.  The instruction set reference I have for Rabbit 4000/5000 assembly says `ld` (at least with the stack) doesn't support `altd`.  But maybe that was a change on the 6000?

So I just wrote a test program, and found that `altd ld py, (sp+@sp+foo)` works on the Rabbit 4000.  So perhaps it's fine to use that notation throughout, and not limit it to `_RAB6K`.  I'm looking at CSV files used in building the compiler source code, and not seeing anything to differentiate between 4000/6000 with opcodes.  If you know of an opcode added to the 6000 and not present on the 4000, then I can look at its entry in those files to try to understand the notation.  But even in looking through compiler source, I'm not seeing anything to indicate that the 6000 added new instructions or changed the behavior of existing instructions.

In addition, I have a spreadsheet of Rabbit 6000 instructions that indicates `altd` affects the destination register of `ld pd,(SP+n)`.  All of the `ld pd,X` and `ldl pd,X` instructions, except for `ld pd,(HTR+HL)` indicate that they work with `altd`.  I don't have a similar sheet for the Rabbit 4000.